app/src/main/res/values/strings.xml

```diff
@@ -132,6 +132,10 @@
     <string name="navigation_drawer_support_project">Support the project</string>
     <string name="navigation_drawer_contribute">How to contribute</string>
     <string name="navigation_drawer_privacy_policy">Privacy policy</string>
+    <string name="account_list_calendar_storage_missing">Calendar system storage missing. Did you uninstall the calendar provider system app?</string>
+    <string name="account_list_contacts_storage_missing">Contacts system storage missing. Did you uninstall the contacts provider system app?</string>
+    <string name="account_list_calendar_storage_deactivated">Calendar system storage is deactivated. Did you disable the calendar provider system app?</string>
+    <string name="account_list_contacts_storage_deactivated">Contacts system storage is deactivated. Did you disable the contacts provider system app?</string>
```

Comment:
I think we should call it "calendar storage" instead of "calendar system storage". Also while I find it good to provide exact messages, I think it's too detailed to differentiate between disabled and not installed. So I suggest:

_Calendar storage missing. Did you disable the calendar storage system app?_

(and silently take uninstalled as part of disabled)

+ same for contacts